### ISSUE TITLE
Is the cityscapes website down?

### ISSUE THREAD

**Issue by zbwxp** [mcordts/cityscapesScripts#141]
Sorry to bother but has the cityscapes website:  
www.cityscapes-dataset.com    
been down for two days already? (Dec 1st 2020 till now)

Is there any scheduled maintenance going on? when will it be back?

Thank you very much.

**Comment by Penitto** [comment]
Yes, it was. There was notification about it

**Comment by mcordts** [comment]
Yes, we had to conduct some maintenance, that took longer than expected. The website is back online. Please let me know if you observe any bugs.

**Comment by Sunyingchun** [comment]
Hi @mcordts  @Penitto  @zbwxp ,When I submit a single zip file named results on the website, which contains the result of instance segmentation on test set according to specifications, the web page shows a red 'Content of zip archive does not match specifications. Not all predictions for all images were found.'. However, I can get AP value in the evalInstanceLevelSemanticLabeling.py file in the same way on the val set. I don't understand why the web page prompts like this. Can you tell me why? 
Thank you very much!
Please forgive my poor English.
